rules/Arguments/ArgumentDefaultValueReplacer.php

```diff
@@ -119,6 +119,8 @@ private function processArgs(
             if (is_array($newArgs)) {
                 $expr->args = $newArgs;
             }
+        } else {
```

Comment:
If we have not replaced the argument, return null to indicate that no change has been made